### ISSUE TITLE
bean-web does not watch all the included files, just the top-level one

### ISSUE THREAD

**Issue by blais** [beancount/beancount#25]
**[Original report](https://bitbucket.org/blais/beancount/issue/25) by Martin Blais (Bitbucket: [blais](https://bitbucket.org/blais), GitHub: [blais](https://github.com/blais)).**

----------------------------------------

- BUG: When multiple files have been loaded, have bean-web watch all the
    loaded files.

       "One question: previously, if I wrote to the beancount file that a
    bean-web process was already using, and if I hit refresh, bean-web noticed
    the update to the file and re-read. This still works now, but only on the
    including file, not on the included files. Is this a pain to fix, or
    relatively easy?"

**Comment by blais** [comment]
**Original comment by Justus Pendleton (Bitbucket: [hoostus](https://bitbucket.org/hoostus), GitHub: [hoostus](https://github.com/hoostus)).**

----------------------------------------

This appears to have been fixed at some point, so I believe the issue can be closed.

I created sample beanfile with multiple includes and bean-web does the right thing.

A quick glance at loader.py looks like it is correctly taking all included files into account, as well, when determining whether a reload is necessary.
